libnetwork/ipams/null/null.go

```diff
@@ -23,11 +23,11 @@ func (a *allocator) GetDefaultAddressSpaces() (string, string, error) {
 	return defaultAddressSpace, defaultAddressSpace, nil
 }
 
-func (a *allocator) RequestPool(addressSpace, pool, subPool string, _ map[string]string, v6 bool) (string, *net.IPNet, map[string]string, error) {
+func (a *allocator) RequestPool(addressSpace, preferredPool, subPool string, _ map[string]string, v6 bool) (string, *net.IPNet, map[string]string, error) {
```

Comment:
I think we should re-use `requestedPool` / `requestedSubPool` here to stay consistent with the interface and default IPAM driver.